spec.md

```diff
@@ -202,29 +202,45 @@ help intermediate gateways determine how to route the events.
 
 ### source
 * Type: `URI-reference`
-* Description: This describes the event producer. Often this will include
+* Description: This identifies the event producer. Often this will include
   information such as the type of the event source, the organization
   publishing the event, the process that produced the event, and some unique
-  identifiers. The exact syntax and semantics behind the data encoded in the URI
-  is event producer defined.
+  identifiers.
+
+  The exact syntax of `source`, and the scope in which it is unique,
+  depends on the application. Applications range from a single service
```

Comment:
I don't think we should talk about "application" here, but only about event producer.

See https://github.com/cloudevents/spec/blob/master/primer.md#design-goals especially `and later can be connected to create new applications`.

An (implementation of an) event producer MAY be part of many applications. The author of the event producer may not know what applications it will be part of after a few years.

Reply:
resolving, please check latest and re-raise any issues.